.gitignore

```diff
@@ -56,3 +56,8 @@ build*/
 _*/
 
 rtl_433_tests
+
+.idea
```

Comment:
Maybe you should add this to your own global gitignore instead of adding it in here. Also the .DS_STORE can be in there. In the original gitignore file the build*/ folder is already ignored, so you could do `mkdir build && cd build && cmake .. && make` instead of making a cmake-build-debug folder. Just some minor comments, but they avoid cluttering this file for every project you want to contribute to.

Reply:
> Maybe you should add this to your own global gitignore instead of adding it in here.

That is a good idea although I thought that others might benefit from the addition of the default ignores for a fairly popular IDE. Are you familiar with IdeaLabs' CLion?

> Also the .DS_STORE can be in there. 

Great... but who died and made you queen?

> In the original gitignore file the build*/ folder is already ignored, so you could do `mkdir build && cd build && cmake .. && make` instead of making a cmake-build-debug folder.

Thanks. I am quite familiar with the build structure of this project. I fixed it back in 50cfd95da14b37678326fdf17fe2ec18213bfb3c. That folder, as mentioned above, is a default folder created by the IDE.

> Just some minor comments, but they avoid cluttering this file for every project you want to contribute to.

Are you really worried about the addition of 4 lines to a file that no one really reads or edits except for when they want to add lines to it they think might benefit others?

Reply:
Please try a friendly and open discussion. Handling your IDE excludes in a global setting does indeed seem apropriate -- with the benefit to get it for free in every repo you use. (btw. that IDE isn't free (FOSS), so it's likely not on most peoples shortlist.)

Reply:
Where was I unfriendly? I replied with humor to what seemed a rather condescending comment. Maybe the humor got lost...

As for the proposed changes, I don't really care one way or the other. I was just trying to help out.

[CLion actually is free (as are all other JetBrains tools) for FOSS projects like rtl_433](https://www.jetbrains.com/store/terms/comparison.html).